lldb/source/Commands/Options.td

```diff
@@ -744,6 +744,10 @@ let Command = "process continue" in {
     Arg<"BreakpointIDRange">, Desc<"Specify a breakpoint to continue to, temporarily "
     "ignoring other breakpoints.  Can be specified more than once.  "
     "The continue action will be done synchronously if this option is specified.">;
+  def thread_continue_forward : Option<"forward", "F">, Group<3>,
```

Comment:
I don't think it's necessary to have 2 options for this, I'd just keep the reverse option and make it default to false. That would simplify the logic in the CommandObject as well.

Reply:
Processes have a persistent base direction. `--forward` forces forward, `--reverse` forces reverse, and neither means continue in the current dirction.